ArchUnitNET/Fluent/PlantUml/GivenPlantUmlFluentComponentDiagram.cs

```diff
@@ -24,9 +24,9 @@ public string AsString(RenderOptions renderOptions = null)
             return _fluentComponentDiagramCreator.Builder.AsString(renderOptions);
         }
 
-        public void WriteToFile(string path, RenderOptions renderOptions = null, bool overwrite = true)
+        public void WriteToFile(string path, RenderOptions renderOptions = null)
```

Comment:
This looks like a breaking change to the API. Is this method already released?

Reply:
I moved this "overwrite" to RenderOptions. The method itself has not yet been used anywhere.